internal/configs/version1/template_helper.go

```diff
@@ -14,18 +14,42 @@ func trim(s string) string {
 	return strings.TrimSpace(s)
 }
 
-// makeLocationPath takes a string representing a location path
-// and a map representing Ingress annotations.
+// makeLocationPath takes location and Ingress annotations and returns
+// modified location path with added regex modifier or the original path
+// if no path-regex annotation is present in ingressAnnotations
+// or in Location's Ingress.
+//
+// Annotation 'path-regex' set on the Minion takes higher priority comparing
+// to the annotation set on the master (in Master-Minion Ingress setup).
```

Comment:
```suggestion
// Annotation 'path-regex' set on a Minion Ingress will take priority over
// the annotation set on the master (in Master-Minion Ingress setup).
```

Reply:
Slight re-wording. The phrase `takes higher priority` makes me think that this is a priority that can be adjusted.

Reply:
Applied change in the new commit, thanks @shaun-nx !